kiosk/src/Components/AddingGame.tsx

```diff
@@ -47,6 +51,22 @@ const AddingGame: React.FC<IProps> = ({ kiosk }) => {
         return true;
     }
 
+    const getGameNames = (gameList: GameData[]): string[] => {
+        const gameNames = [];
+        for (const game of gameList) {
+            gameNames.push(game.name);
+        }
+        return gameNames;
+    }
+
+    const displayGamesAdded = (addedGames: GameData[]): void => {
+        const gameNames = getGameNames(addedGames);
+        const games = gameNames.toString();
+        const notification = `Game(s) [ ${games} ] added to kiosk!`
```

Comment:
- Yes, kiosk implied, I can definitely get rid of it. 
- Yes, "Game(s) can be omitted. I think in general, I was trying to leave no ambiguity with the notification, but it really shouldn't be necessary since arcade games are the only possible thing that can be added to kiosk.
- The brackets were nice to make the list of games stand out the most, but you're right, simplifying it is the best way to give the most attention to the game titles that were added.